generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -199,3 +199,12 @@ class TestClockRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithAsynchronousRocketTiles(8, 3) ++   // Add rational crossings between RocketTile and uncore
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new chipyard.config.AbstractConfig)
+
+// Multiclock Sketch
+class ForcedClockRocketConfig extends Config(
+  new chipyard.config.WithForcedTileFrequency(200) ++
```

Comment:
What is the assumed frequency for the rest of the system? 100Mhz?

Reply:
I believe so, but is that because the Dealiser thing is setting everything to use the `take` frequency of the PBus, which is the only clock in the `subsystem_sbus` clock group which has its `take` parameter set? https://github.com/chipsalliance/rocket-chip/blob/6f0cf5463a0e2942cb49ab69ab048e6ba2b4378c/src/main/scala/subsystem/Configs.scala#L29-L32

What is a reasonable frequency for everything else? 1GHz?